test/integration_test/common_test.go

```diff
@@ -98,9 +101,13 @@ func TestMain(t *testing.T) {
 	}
 	// we can either run cluster domain tests or the other tests
 	// the underlying setup of clusters is different
-	if os.Getenv("ENABLE_CLUSTER_DOMAIN_TESTS") != "" {
-		t.Run("ClusterDomains", testClusterDomains)
+	if enabled, err := strconv.ParseBool(os.Getenv(enableClusterDomainTests)); err == nil {
+		if enabled {
```

Comment:
If enabled is false this won't run any test

Reply:
Fixed.